internal/db/testutils.go

```
@@ -0,0 +1,31 @@
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+func SetupTestDB(t *testing.T) *sql.DB {
+	t.Helper()
+	db, err := NewDatabase(":memory:")
+	if err != nil {
+		t.Fatal("failed to init database: " + err.Error())
+	}
+	t.Cleanup(func() {
+		db.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `db.Close` is not checked

(errcheck)